features/steps/steps.py

```diff
@@ -88,6 +109,16 @@ def step_impl(context, order_status):
         )
     )
     expect(found).to_be(True)
+    
+@then('I should see "{order_status}" in the list items results')
+def step_impl(context, order_status):
+    found = WebDriverWait(context.driver, context.WAIT_SECONDS).until(
+        expected_conditions.text_to_be_present_in_element(
+            (By.ID, 'list_item_results'),
+            order_status
+        )
+    )
+    expect(found).to_be(True)
```

Comment:
Instead of adding a new function, take another argument entity here so that it becomes:
```
@then('I should see "{message}" in the "{entity}" results')
def step_impl(context, message, entity):
    found = WebDriverWait(context.driver, context.WAIT_SECONDS).until(
        expected_conditions.text_to_be_present_in_element(
            (By.ID, entity + '_search_results'),
            message
        )
    )
    expect(found).to_be(True)
```

and then rename the tests to say something like this:
```
I should see "CONFIRMED" in the "Order" results

I should see "123" in the "Item" results
```